### ISSUE TITLE
[Callout with media] Text not using color tokens and failing on dark themes

### ISSUE THREAD

**Issue by shixiedesign** [carbon-design-system/carbon-for-ibm-dotcom#5676]
**Headline**, **body copy**, and **caption** in this component are not using color tokens. 

### Current:
![Mar-30-2021 15-48-46.gif](https://images.zenhubusercontent.com/5be09c3a6102900c6d16d991/b1958ade-f9a1-4035-92cd-9047e7935bca)

### Expecting: headline, body copy, and caption all use color `$inverse-01`
![image.png](https://images.zenhubusercontent.com/5be09c3a6102900c6d16d991/5c9a88d6-775c-40fb-bfa7-02de1025c336)

**Comment by RobertaJHahn** [comment]
The merged PR includes discussions that brought about this new issue.  Reopening this issue to finish the work.

**Comment by guilhermelMoraes** [comment]
Fixed by @ariellalgilmore on #5828
